### ISSUE TITLE
npm run dev gives me the below screen capture - "Server Error Error: Not supported"

### ISSUE THREAD

**Issue by eusholli** [timlrx/tailwind-nextjs-starter-blog#214]
Have you seen this before?    I think it is to do with ESM support but I cannot resolve and I do not understand how others have it working.

This is what i immediately see from visiting http://localhost:3000 

![image](https://user-images.githubusercontent.com/158977/131259480-e9f54028-959e-4b59-979d-4d602d440d25.png)

**Comment by timlrx** [comment]
Hi Geoff, is it a newly cloned instance, or did you upgrade it from an older version? The latest version supports [esm](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/next.config.js#L11) and uses esm only remark / rehype packages. Also, what version of node are you running it on?

**Comment by eusholli** [comment]
Hi Timothy
thank you for coming back and your questions resolved the issue and I feel stupid.  I thought my node installation was auto-updating but it was not.  The version I was running was v12.16.3.  I actually rolled back to v12.1 yesterday based on another post I saw but that did not fix the issue. Upgrading nodeJS to the latest (at time of writing v14.17.5 resolved all issues.
I thought it strange but this was the only thing I did not check.
thank you. And thank you for the template
